lib/Elastica/Result.php

```diff
@@ -197,18 +197,10 @@ public function getDocument()
         $doc = new Document();
         $doc->setData($this->getSource());
         $hit = $this->getHit();
-        if ($this->hasParam('_source')) {
```

Comment:
I thought that in previous version of PHP this was throwing a notice, but I just tried it in PHP 5.4 in it looks fine.